src/chains/MatchAllRequestsProcessorChain.ts

```diff
@@ -0,0 +1,14 @@
+import ProcessorChain, { IRequestMatchingProcessorChain } from './ProcessorChain';
+import { ErrorHandlingRequestProcessor } from '../interfaces';
+
+export class MatchAllRequestsProcessorChain extends ProcessorChain implements IRequestMatchingProcessorChain {
+
+   public constructor(subprocessors: ErrorHandlingRequestProcessor[]) {
```

Comment:
This constructor isn't necessary if all it's doing is calling `super`, unless there's another reason for having it I'm not thinking of.

Reply:
Wasn't sure if you missed this comment ^^